### PR TITLE
sgp4 ready for review

### PR DESCRIPTION
**Pull request by jochym**
Python implementation of most recent version of the SGP4 satellite tracking algorithm.
This is a dependency of the skyfield library (WIP presently).
I believe the recipe is ready for inspection and possibly merging.

### CODE REVIEW

File: recipes/sgp4/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  imports:
+    - sgp4
+    - sgp4.tests
+  commands:
+    - python -m unittest -v sgp4.tests
+
+about:
+  home: https://github.com/brandon-rhodes/python-sgp4
+  license: MIT
+#  license_file: LICENSE
```
Comment (scopatz) [inline recipes/sgp4/meta.yaml]: Any reason not to add the licence file?

Comment (jochym) [inline recipes/sgp4/meta.yaml]: It is not in original tarball and got added in the dev tree. I can add it manually if it is important.

Comment (jochym) [inline recipes/sgp4/meta.yaml]: @scopatz Maybe I am stupid but for the life of me I cannot figure out the way to just add the file manually - like I did in 40f2ae7. It did not get recognized by the build system. I have tried to add it to the package by patching but that adds the file to sgp4 package directory and this is also not recognized. Can you help me with this?

File: recipes/sgp4/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  imports:
+    - sgp4
+    - sgp4.tests
+  commands:
+    - python -m unittest -v sgp4.tests
+
+about:
+  home: https://github.com/brandon-rhodes/python-sgp4
+  license: MIT
+  license_file: sgp4/LICENSE
```
Comment (ocefpaf) [inline recipes/sgp4/meta.yaml]: I believe that conda, when parsing `license_file` for you, enter the source directory. So maybe `license_file: LICENSE` would suffice. Can you test it?

Comment (jochym) [inline recipes/sgp4/meta.yaml]: I will, but I have low expectations. I believe I have done just this in 40f2ae7. Added LICENSE to recipe directory and to license_file. But I will check anyway.

Comment (ocefpaf) [inline recipes/sgp4/meta.yaml]: Let me check this closer. Now I see that you are adding as a patch... Hold on...

Comment (jochym) [inline recipes/sgp4/meta.yaml]: OK. Holding my horses ;)

**Pull request merged**